spec/appMenu.spec.ts

```diff
@@ -0,0 +1,358 @@
+import { AppMenu, menuSections } from '../src/app/app-menu';
+import { autoLaunchInstance } from '../src/app/auto-launch-controller';
+import { config } from '../src/app/config-handler';
+import { exportCrashDumps, exportLogs } from '../src/app/reports-handler';
+import { updateAlwaysOnTop } from '../src/app/window-actions';
+import { windowHandler } from '../src/app/window-handler';
+import * as envMock from '../src/common/env';
+import { logger } from '../src/common/logger';
+import { dialog, session, shell } from './__mocks__/electron';
+
+jest.mock('../src/app/reports-handler', () => {
+    return {
+        exportLogs: jest.fn(),
+        exportCrashDumps: jest.fn(),
+    };
+});
+
+jest.mock('../src/common/env', () => {
+    return {
+        isWindowsOS: false,
+        isMac: true,
+    };
+});
+
+jest.mock('../src/app/window-actions', () => {
+    return {
+        updateAlwaysOnTop: jest.fn(),
+    };
+});
+
+jest.mock('../src/app/auto-launch-controller', () => {
+    return {
+        autoLaunchInstance: {
+            disableAutoLaunch: jest.fn(),
+            enableAutoLaunch: jest.fn(),
+        },
+    };
+});
+
+jest.mock('../src/app/config-handler', () => {
+    return {
+        config: {
+            getConfigFields: jest.fn(() => {
+                return {
+                    minimizeOnClose: true,
+                    launchOnStartup: true,
+                    alwaysOnTop: true,
+                    isAlwaysOnTop: true,
+                    bringToFront: true,
+                    memoryRefresh: true,
+                };
+            }),
+            getGlobalConfigFields: jest.fn(() => {
+                return {
+                    devToolsEnabled: true,
+                };
+            }),
+            updateUserConfig: jest.fn(),
+        },
+    };
+});
+
+jest.mock('../src/app/window-handler', () => {
+    return {
+        windowHandler: {
+            createMoreInfoWindow: jest.fn(),
+        },
+    };
+});
+
+jest.mock('../src/common/logger', () => {
+    return {
+        logger: {
+            error: jest.fn(),
+            info: jest.fn(),
+        },
+    };
+});
+
+describe('app menu', () => {
+    let appMenu;
+    const updateUserFnLabel = 'updateUserConfig';
+    const item = {
+        checked: true,
+    };
+
+    const findMenuItemBuildWindowMenu = (menuItemLabel: string) => {
+        // tslint:disable-next-line:no-shadowed-variable
+        return appMenu.buildWindowMenu().submenu.find((item) => {
```

Comment:
@vrbsm You can just use a different name like `menuItem` instead of disabling `tslint`